doc/README.md

```diff
@@ -18,6 +18,7 @@ APIs:
 * [Repositories](repos.md)
   * [Contents](repo/contents.md)
   * [Releases](repo/releases.md)
+  * [Tags](repo/tags.md)
```

Comment:
this is the wrong place for it. ``* [Assets](repo/assets.md)`` is a sub-point of ``Releases``